apis/ecr/v1alpha1/repository_types.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	runtimev1alpha1 "github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+)
+
+// RepositoryParameters define the desired state of an AWS Elastic Container Repository
+type RepositoryParameters struct {
+
+	// Region is the region you'd like your Certificate to be created in.
```

Comment:
```suggestion
	// Region is the region you'd like your Repository to be created in.
```

I am guessing this is the intended?

Reply:
yepp good ole copy paste